### PR TITLE
feat(template/vanilla): use a namespace 🏎 

### PR DESCRIPTION
**Pull request by qbzzt**

### CODE REVIEW

**Comment by qbzzt** [comment]
I don't know how to fix the test, but I released a snapshot and it works.

**Comment by holic** [comment]
> I don't know how to fix the test, but I released a snapshot and it works.

need to run codegen!

**Comment by qbzzt** [comment]
> > I don't know how to fix the test, but I released a snapshot and it works.
> 
> need to run codegen! `pnpm run build` from `packages/contracts`

Thank you! I worked here, and I'm now doing the same on the other PRs.

**Comment by holic** [comment]
any chance we can do all the templates in one PR?

File: templates/vanilla/packages/contracts/mud.config.ts [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import { defineWorld } from "@latticexyz/world";
 
 export default defineWorld({
+  namespace: "counter",
```
Comment (alvrs) [inline templates/vanilla/packages/contracts/mud.config.ts:4]: thoughts about using `home` as default namespace in templates? cc @holic

**Comment by qbzzt** [comment]
How about `main`?

**Comment by qbzzt** [comment]
Closed in favor of a single PR with all the templates.

**Pull request closed without merging**